pgrx-examples/strings/src/lib.rs

```diff
@@ -33,11 +33,6 @@ fn append(mut input: String, extra: &str) -> String {
     input
 }
 
-#[pg_extern]
-fn split(input: &'static str, pattern: &str) -> Vec<&'static str> {
-    input.split_terminator(pattern).collect()
-}
-
```

Comment:
Why was this removed?  And what *should* the function signature look like?

Reply:
I'm guessing it's the `'static` that's objectionable here.

Reply:
Correct.

Reply:
```rust
#[pg_extern]
fn split<'a>(input: &'a str, pattern: &str) -> Vec<&'a str> {
    input.split_terminator(pattern).collect()
}
```

Is that the new form?  Should we just update that example rather than removing it?

Reply:
Sure. It seemed slightly redundant with the next example, is probably why I deleted it.